testsuite/MDAnalysisTests/coordinates/test_chemfiles.py

```diff
@@ -236,7 +242,10 @@ def test_write_atom_group(self, tmpdir):
             check = mda.Universe(outfile)
             assert check.trajectory.ts.n_atoms == group.n_atoms
 
-    @pytest.mark.xfail  # Issue #3798 - float writing not consisten
+    @pytest.mark.skipif(
```

Comment:
This really isn't ideal, if anything we really shouldn't have code that does an indirect explicit float comparison like this.

The main question is - was this change in writing behaviour intended?

Reply:
Yes, the change was intended on chemfiles side. I can use a different format (maybe a binary format?) than LAMMS data files to protect such small changes, since the purpose of this tests it to check velocities are passed & used by chemfiles; not the actual output.

Reply:
IIRC its closer to how ASE writes data files but not LAMMPs itself does, but moot point anyway as this is about chemfiles support.

Reply:
As long as it's an intended change in writing behaviour, and that chemfiles can guarantee there won't be any arbitrary (i.e. non-intentional) changes in this writing behaviour, then I'm happy.

I know I'm being a pedantic here, but it's better being clear about how things are intended to work now than to have to come back here in 3-6 months.

Reply:
I totally understand wanting to be clear! We have very similar tests on the chemfiles side (e.g. https://github.com/chemfiles/chemfiles/blob/387d4920b7eb5107ccd6050da238f82dacf1c85b/tests/formats/lammps-data.cpp#L115), so there should not be any unintentional changes here.

There might still be intentional changes that would introduce extra "noise" on the MDA side (although I can be more mindful of this), so I'll still try to write a more stable test here, maybe using a different format.

Reply:
Extra noise isn't as much of an issue (to me at least) to be honest (although more stable tests are always appreciated, that can be future work if you want).

Reply:
I've updated the test to use a NetCDF file instead, which can easily be checked with scipy